src/main/java/com/java/study/group/librarysystem/controller/CreateCourseController.java

```diff
@@ -0,0 +1,26 @@
+package com.java.study.group.librarysystem.controller;
```

Comment:
Delete this class as there is already a controller created for the register course details.